### ISSUE TITLE
Add an FAQ and/or high-level overview of how to integrate TUF

### ISSUE THREAD

**Issue by vladimir-v-diaz** [theupdateframework/python-tuf#486]
We are often asked the same questions regarding delegations, how to organize TUF metadata, key management, etc.  There should be an FAQ page for these sorts of questions.

The [Survivable Key Compromise](https://github.com/theupdateframework/tuf/pull/485) paper provides an overview of TUF's design, and the specification and other documents available here capture most of what it contains.  However, the integration side of things is not adequately explained outside of the [Diplomat](https://github.com/theupdateframework/tuf/blob/develop/docs/papers/protect-community-repositories-nsdi2016.pdf) and Mercury papers.  The PEP 458 document is also a good resource for how to integrate TUF with an existing software updater and community repository.

We should provide a *condensed*, high-level document that contains the information in the papers and documents that have been written since the Survivable Key Compromise paper.

**Comment by vladimir-v-diaz** [comment]
We are working on an FAQ page for the website.  See https://github.com/theupdateframework/theupdateframework.github.io/pull/29.  There should also be a document that captures more of the recent developments from the Mercury and Diplomat papers.

**Comment by MVrachev** [comment]
I don't think this is a `good-first-issue`. To write such a page one should understand how TUF works.

**Comment by lukpueh** [comment]
Duplicate of #2452
